deep_agent/aegra/entrypoint.py

```
@@ -64,8 +70,20 @@ def validate_config_mount() -> None:
 
-def start_agent() -> None:
-    """Start the agent runtime after config validation."""
+def _read_mode() -> str:
+    """Read agent mode from runtime/agent.yaml, defaulting to 'server'."""
+    agent_yaml = CONFIG_PATH / "runtime" / "agent.yaml"
+    if not agent_yaml.is_file():
+        return "server"
+    try:
+        data = yaml.safe_load(agent_yaml.read_text()) or {}
+        return str(data.get("mode", "server")).strip().lower()
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)